esp-hal-embassy/README.md

```diff
@@ -0,0 +1,35 @@
+# esp-hal-embassy
+
+[![Crates.io](https://img.shields.io/crates/v/esp-hal-embassy?labelColor=1C2C2E&color=C96329&logo=Rust&style=flat-square)](https://crates.io/crates/esp-hal-embassy)
+[![docs.rs](https://img.shields.io/docsrs/esp-hal-embassy?labelColor=1C2C2E&color=C96329&logo=rust&style=flat-square)](https://docs.rs/esp-hal-embassy)
+![MSRV](https://img.shields.io/badge/MSRV-1.76-blue?labelColor=1C2C2E&style=flat-square)
+![Crates.io](https://img.shields.io/crates/l/esp-hal-embassy?labelColor=1C2C2E&style=flat-square)
+[![Matrix](https://img.shields.io/matrix/esp-rs:matrix.org?label=join%20matrix&labelColor=1C2C2E&color=BEC5C9&logo=matrix&style=flat-square)](https://matrix.to/#/#esp-rs:matrix.org)
+
+[Embassy] support for `esp-hal`.
```

Comment:
The `README.md` could definitely use some expansion, however I'm not really sure what should be included here and what belongs in documentation.

Reply:
Shall we add some links to the embassy [book](https://embassy.dev/book/index.html) or to [website](https://embassy.dev/)? They are just 1 click further from the repo, but it might be useful.